numpy/tests/test_ctypeslib.py

```diff
@@ -21,11 +21,12 @@
 except ImportError:
     _HAS_CTYPE = False
 
+
+@pytest.mark.skipif(not _HAS_CTYPE,
```

Comment:
See above, ctypes should be available in the Python versions we support.

Reply:
Discussed below